cmd/transform/create.go

```diff
@@ -49,7 +49,7 @@ func newCreateCommand() *cobra.Command {
 			log.Debug("Transform", "transform", transform)
 
 			if transform.GetName() == "" {
-				return fmt.Errorf("the transform must have a name")
+				return fmt.Errorf("The transform must have a name")
```

Comment:
So many of these changes are amazing, and I love that you have gone through here so thoroughly.

In this case specifically go kinda throws a fit, they have a linting message that all error strings should start with a lowercase character

https://google.github.io/styleguide/go/decisions.html#error-strings